### PR TITLE
[zh-cn] sync feature-gates-removed.md

### PR DESCRIPTION
**Pull request by gaogao101**
Signed-off-by: Gao Qian <gaoq.jy@fujitsu.com>

### CODE REVIEW

**Review by my-git9 — COMMENTED** [review]
/lgtm

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates-removed.md [hunk 1]
```diff
@@ -790,6 +803,9 @@ In the following table:
 
 - `PodDisruptionBudget`：启用 [PodDisruptionBudget](/zh-cn/docs/tasks/run-application/configure-pdb/) 功能特性。
 
+- `PodOverhead`：启用 [PodOverhead](/zh-cn/docs/concepts/scheduling-eviction/pod-overhead/)
+  特性以考虑 Pod 开销。
```
Comment (my-git9) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates-removed.md:807]:
```suggestion
  特性以计算 Pod 开销。
```

**Review by my-git9 — CHANGES_REQUESTED** [review]

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates-removed.md [hunk 2]
```diff
@@ -540,6 +548,8 @@ In the following table:
 - `CSIVolumeFSGroupPolicy`：允许 CSIDrivers 使用 `fsGroupPolicy` 字段。
   该字段能控制由 CSIDriver 创建的卷在挂载这些卷时是否支持卷所有权和权限修改。
 
+- `CSRDuration`：允许客户端请求通过 Kubernetes CSR API 签署的证书的有效期限。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates-removed.md:551]:
```suggestion
- `CSRDuration`：允许客户端为通过 Kubernetes CSR API 签署的证书申请有效期限。
```

**Pull request merged**